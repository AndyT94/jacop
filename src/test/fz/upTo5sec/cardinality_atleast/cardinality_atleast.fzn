var 1..1: nvar:: output_var;
array [1..3] of var int: variables:: output_array([1..3]) = [3,3,8];
array [1..2] of var int: values:: output_array([1..2]) = [3,8];
solve  satisfy;
