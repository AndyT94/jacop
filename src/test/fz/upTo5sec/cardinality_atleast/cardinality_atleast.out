nvar = 1;
variables = array1d(1..3, [3, 3, 8]);
values = array1d(1..2, [3, 8]);
----------